doc/src/manual/arrays.md

```diff
@@ -182,7 +182,7 @@ julia> sum(1/n^2 for n=1:1000)
 When writing a generator expression with multiple dimensions inside an argument list, parentheses
 are needed to separate the generator from subsequent arguments:
 
-```julia
+```julia-repl
```

Comment:
Documenter can't handle syntax errors as far as I can tell (at the moment, should probably be fixed).